sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/LockRenewalOperation.java

```diff
@@ -191,13 +199,20 @@ public void close() {
             .thenReturn(Flux.create(s -> s.next(interval)))))
             .takeUntilOther(cancellationSignals)
             .flatMap(delay -> {
-                logger.info("token[{}]. now[{}]. Starting lock renewal.", lockToken, OffsetDateTime.now());
+                logger.atInfo()
+                     // TODO (limolkova) do we really need it? log timestamp should be already there
+                    .addKeyValue("now", OffsetDateTime.now())
```

Comment:
why do we need a timestamp in the message?

Reply:
I think we can remove this explicit logging for the time. IIRC, it was to see how long a renewal operation took... if it took too long, by the time we renewed the token, it could have expired.